src/pmsearch/pmsearch_daily.sh

```
@@ -0,0 +1,240 @@
+    }
+    if (name in metric_indom)
+	printf "@ M %s\t%s\t%s\n", name, metric_indom[name], oneline
+    else
+	printf "@ M %s\t\t%s\n", name, oneline
+    next
+}
+/^Help:$/ { next }
+/^Full Help: Error:/ { skip = 1; next }
+skip { next }
+{ print }
+' "$tmp/metric_indom" - > $tmp/helptext
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)